src/adapter/src/coord.rs

```diff
@@ -1269,7 +1274,8 @@ impl<S: Append + 'static> Coordinator<S> {
             })
             .collect::<Vec<_>>();
         let num_updates = appends.len();
-        self.controller.storage_mut().append(appends).await.unwrap();
+        // Note: Do not await the result; let it drop (as we don't need to block on completion)
+        self.controller.storage_mut().append(appends);
```

Comment:
If we're now doing these asynchronously now, should we still be queuing the updates and sending them in batches? It seems fine to fire them all at once and forget about them.

Reply:
If we do that I should edit it a bit to consolidate the writes in the tokio task. But I wanted to randomize things in the coordinator as little as possible.